frontend/app/components/account-form.js

```diff
@@ -30,6 +31,8 @@ export default class AccountForm extends BaseFormComponent {
       AccountValidations
     );
 
+    this.presetTeamAndFolder()
```

Comment:
Maybe do the `if (this.isNewRecord)` here. Otherwise, this method would be called `presetTeamAndFolderIfIsNewRecord` right? :P

Reply:
Sure.